doc/manuals/admin/build_source.md

```diff
@@ -85,16 +86,29 @@ for details. Recommended version is 3.6 (although 3.2 and 3.4 should also work f
 
 The Orion Context Broker comes with a suite of functional, valgrind and end-to-end tests that you can also run, following the following procedure (optional):
 
+* To install mongodb-org-shell using yum, create a /etc/yum.repos.d/mongodb.repo file.
+
+        [mongodb-org-3.6]
+        name=MongoDB Repository
+        baseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.6/x86_64/
+        gpgcheck=1
+        enabled=1
+        gpgkey=https://www.mongodb.org/static/pgp/server-3.6.asc
+
 * Install the required tools:
 
-        sudo yum install python python-flask pyOpenSSL curl nc mongodb-org-shell valgrind bc
+        sudo yum install python curl nc mongodb-org-shell valgrind bc python-devel libffi-devel python-pip
```

Comment:
I understand your approach and I think is better to use pip dependencies (installed in a virtualenv) instead that system modules from python-flask and pyOpenSSL. Thus, I understand the need for python-devel and python-pip. However, what about libffi-devel? What's it? Could you elaborate, please?

Reply:
The pyOpenSSL requires the cryptography as a dependncy.
In the case of x86_64, the crypotgrapy is installed from a .whl file. 
```
Collecting cryptography>=2.8
  Downloading cryptography-2.8-cp27-cp27mu-manylinux2010_x86_64.whl (2.3 MB) 
```
In the case of aarch64, the crypotgrapy is built from a source code. The libffi-devel is needed as a dependcy library for build.
```
Collecting cryptography>=2.8
  Downloading cryptography-2.8.tar.gz (504 kB) 
```

The details are the following:
```
Collecting cryptography>=2.8
  Using cached cryptography-2.8.tar.gz (504 kB)
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python2 /usr/lib/python2.7/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-aYE2lL/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- 'setuptools>=40.6.0' wheel 'cffi>=1.8,!=1.11.3; platform_python_implementation != '"'"'PyPy'"'"''
       cwd: None
  Complete output (88 lines):
  DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
  Collecting setuptools>=40.6.0
    Using cached setuptools-44.0.0-py2.py3-none-any.whl (583 kB)
  Collecting wheel
    Using cached wheel-0.34.2-py2.py3-none-any.whl (26 kB)
  Collecting cffi!=1.11.3,>=1.8
    Using cached cffi-1.14.0.tar.gz (463 kB)
  Collecting pycparser
    Using cached pycparser-2.19.tar.gz (158 kB)
  Installing collected packages: setuptools, wheel, pycparser, cffi
      Running setup.py install for pycparser: started
      Running setup.py install for pycparser: finished with status 'done'
      Running setup.py install for cffi: started
      Running setup.py install for cffi: finished with status 'error'
      ERROR: Command errored out with exit status 1:
       command: /usr/bin/python2 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-1i9FKC/cffi/setup.py'"'"'; __file__='"'"'/tmp/pip-install-1i9FKC/cffi/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-iJbp8Y/install-record.txt --single-version-externally-managed --prefix /tmp/pip-build-env-aYE2lL/overlay --compile --install-headers /tmp/pip-build-env-aYE2lL/overlay/include/python2.7/cffi
           cwd: /tmp/pip-install-1i9FKC/cffi/
      Complete output (68 lines):
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      No package 'libffi' found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      No package 'libffi' found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      No package 'libffi' found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      No package 'libffi' found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      No package 'libffi' found
      unable to execute gcc: No such file or directory
      unable to execute gcc: No such file or directory
 ```

Reply:
Thanks for the clarification!

I see this part has changed in the PR. I'll provide additional feedback on the new version. NTC with regards this comment thread.